docs/testing-components/unit-testing/index.md

```diff
@@ -115,41 +115,26 @@ and outputs we can test basically any JavaScript function that returns a value.
 
 ## Testing React
 
-To test react we use [Enzyme](https://enzymejs.github.io/enzyme/) plus other tools you can find out about [here](../test-driven-development/index.md).
+To test react we use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) plus other tools you can find out about [here](../test-driven-development/index.md).
 
 ```js
 const Text = (props) => {
 	return <p>{props.content}</p>;
```

Comment:
you can modify this component so  the < p > can receive the rest of the props

Reply:
Done